packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 3.4.0 - Feb 21, 2022
+Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+
+Enhancement:
+- Improved local debug experience, more light weighted and more graceful.
+- Improved the Teams app authentication support.
```

Comment:
We just fixed a bug in tab scaffolding template, we may remove this and see whether we need to tell users what bugs we fixed in the changelog.